simple/stats/variable_per_row_importer.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2024 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from csv import DictReader
+import logging
+import random
+
+from stats import constants
+from stats.data import Observation
+from stats.db import Db
+from stats.importer import Importer
+from stats.nodes import Nodes
+from stats.reporter import FileImportReporter
+from util.filehandler import FileHandler
+
+_COLUMNS = [
+    constants.COLUMN_ENTITY, constants.COLUMN_VARIABLE, constants.COLUMN_DATE,
+    constants.COLUMN_VALUE
+]
+_DEFAULT_COLUMN_MAPPINGS = {x: x for x in _COLUMNS}
+
+
+class VariablePerRowImporter(Importer):
+  """Imports a single observations input file where variables are specified in rows (aka "SVObs").
```

Comment:
The class name does not seem to match the description? This is also about observation but it contrasts with "ObservationsImporter"?

Reply:
True but `VariablePerRowObservationsImporter` felt too long so went with `VariablePerRowImporter`.